### ISSUE TITLE
when Xtree 2? 😛

### ISSUE THREAD

**Issue by oscaramos** [Rodp63/Xtree#1]

**Comment by rodp63** [comment]
@oscaramos probably soon! Any new features you want to see?

**Comment by oscaramos** [comment]
@rodp63 could you please add something that gets the nicest girls around a radius of X kilometers? ☺️

**Comment by rodp63** [comment]
@oscaramos yeah, our next step will definitely be to make it general purpose and expand the use cases (including your nice girls).
